content/ch-quantum-hardware/ac-Stark-shift.ipynb

```diff
@@ -46,10 +46,12 @@
     "# Importing standard Qiskit libraries and configuring account\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.                                        width = drive_with,
</pre><p>You can just set <code>risefall_sigma_ratio=2</code> rather than computing width manually. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>OK. I will change to use risefall_sigma_ratio</p>